### PR TITLE
Fix small mistakes in Twitter example on quickstart page

### PR DESCRIPTION
**Pull request by daniula**
close frida/frida-website#41

### CODE REVIEW

File: _docs/quickstart.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ For the impatient, here's how to do function tracing with Frida:
 
 {% highlight bash %}
 ~ $ sudo pip install frida
-~ $ frida-trace -i "recv*" -i "read*" Twitter
+~ $ frida-trace -i "recv*" -i "read*" com.twitter.android
```
Comment (oleavr) [inline _docs/quickstart.md]: Perhaps we should leave this as “Twitter” so it doesn't sound like an Android-specific example? (You'd typically use the `-U` switch if the process was on a connected Android device.)

**Comment by oleavr** [comment]
Thanks! Just one nitpick and this is good to go.

File: _docs/quickstart.md (hunk 1 above)
Comment (daniula) [inline _docs/quickstart.md]: Ah, that's true. What about `*twitter*` ? It should work on all platforms.

Comment (oleavr) [inline _docs/quickstart.md]: Sounds great!

Comment (daniula) [inline _docs/quickstart.md]: Fixed

**Comment by oleavr** [comment]
Sweet. Thanks! :+1:

**Pull request merged**